<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="4" rowspan="6">
    <widget class="QGroupBox" name="grpPreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Export Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="previewText"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnToClipboard">
          <property name="text">
           <string>Copy to Clipboard</string>
          </property>
          <property name="icon">
           <iconset theme="edit-copy"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveAs">
          <property name="text">
           <string>Save As...</string>
          </property>
          <property name="icon">
           <iconset theme="document-save-as"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="grpReportType">
     <property name="title">
      <string>Report Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioTimesCsv">
        <property name="text">
         <string>Times as CSV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioHistoryCsv">
        <property name="text">
         <string>History as CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioEventLogCsv">
        <property name="text">
         <string>Event Log as CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTimesText">
        <property name="text">
         <string>Times as Text</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="grpDateRange">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>An inclusive date range for reporting on time card history.  Not enabled when reporting on totals.</string>
     </property>
     <property name="title">
      <string>Date Range</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>From:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="KDateComboBox" name="dtFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>To:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KDateComboBox" name="dtTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="combodecimalminutes">
     <item>
      <property name="text">
       <string comment="format to display times">Decimal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hours:Minutes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="combosessiontimes">
     <item>
      <property name="text">
       <string>All Times</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Session Times</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboalltasks">
     <item>
      <property name="text">
       <string>All Tasks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only Selected</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="grpDelimiter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>The character used to separate one field from another in the output.</string>
     </property>
     <property name="title">
      <string>Delimiter</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioComma">
        <property name="text">
         <string>Comma</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioSemicolon">
        <property name="text">
         <string>Semicolon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioOther">
        <property name="text">
         <string comment="user can set an user defined delimiter">Other:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioTab">
        <property name="text">
         <string comment="tabulator delimiter">Tab</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioSpace">
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txtOther">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="quotesLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Quotes:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QComboBox" name="cboQuote">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>All fields are quoted in the output.</string>
     </property>
     <item>
      <property name="text">
       <string>&quot;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>'</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dtFrom</tabstop>
  <tabstop>dtTo</tabstop>
  <tabstop>radioComma</tabstop>
  <tabstop>txtOther</tabstop>
  <tabstop>cboQuote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtOther</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
